src/main/java/org/tikv/common/TiSession.java

```diff
@@ -103,6 +103,18 @@ public TiSession(TiConfiguration conf) {
     warmUp();
     this.circuitBreaker = new CircuitBreakerImpl(conf);
     logger.info("TiSession initialized in " + conf.getKvMode() + " mode");
+    printVersionInfo();
+  }
+
+  private void printVersionInfo() {
```

Comment:
how about refactoring this function to: 
```java
private void getVersionInfo() String {
}
```

then log the info in `TiSession` and add a UT for this function.

Reply:
It's hard to write suck UT to assert the commit hash.

Reply:
Now, I capture these logic into `VersionInfo`, PTAL.